pom.xml

```diff
@@ -16,7 +16,7 @@
   <artifactId>aws-secretsmanager-jdbc</artifactId>
   <packaging>jar</packaging>
   <name>AWS Secrets Manager SQL Connection Library</name>
-  <version>1.0.11</version>
+  <version>1.0.12</version>
```

Comment:
Since we are bumping this, we should bump the example in the readme [here](https://github.com/aws/aws-secretsmanager-jdbc/blob/master/README.md?plain=1#L28)